extensions/blocks/story/player/components/header.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * WordPress dependencies
+ */
+import { createElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { SimpleButton } from './button';
+
+export default function Header( { fullscreen, onExitFullscreen, siteIconUrl, storyTitle } ) {
+	if ( ! fullscreen ) {
+		return null;
+	}
+
+	// default WP logo url relative to story/view.js
+	siteIconUrl = siteIconUrl || '../../../../../../wp-includes/images/w-logo-blue.png';
```

Comment:
This is definitely going to be problematic, since the WordPress install does not always live there. We'd need to fetch the includes directory dynamically, or maybe pull an image that we ship with Jetpack instead? 

As a site owner, I think I'd be more interested in seeing my own site logo there, so maybe that's what we should try to use?

Reply:
You sure the directory structure could be different? This is a relative url so it should find the wp-includes dir relatively from the view.js script for the block. Since it's relative it should work even if wordpress is installed in a subdir. That said, user can always remove or rename the image so it's not super reliable.

Shipping this image with jetpack could make more sense I agree. I'm not sure what we'd want as a default, we need a square image and the logo isn't necessarily one afaik.

Reply:
Actually I'm already resolving this url in the PHP code [here](https://github.com/Automattic/jetpack/pull/16302/files#diff-b7f04975235c31f89dd45854dbc8079fR125) so I can ditch that code.

Reply:
> You sure the directory structure could be different? 

Yeah, basically not all sites have plugins and themes live in `wp-content/`, so the directory structure of the site (and consequently the location of the block files compared to `wp-includes/`) is not always the same.